apps/web/src/components/MarkdownEditor.tsx

```
@@ -1,15 +1,15 @@
 import 'react-mde/lib/styles/css/react-mde-all.css'
-import 'src/components/Fields/styles.css'
+import '@buildeross/ui/styles'
 
 import { getFetchableUrls, uploadFile } from '@buildeross/ipfs-service'
+import { Error } from '@buildeross/ui'
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)